DemoController.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class DemoController : MonoBehaviour
+{
+    public GameObject TalkingHead;
+    public GameObject TalkBubble;
+    public GameObject CloseButton;
+
+    public Button[] TooltipButtons;
+
+    public string AnimationToPlay;
+
+    [Header("Debug & Unit Testing")]
+    public string MyDebugAnimation;

```

Comment:
I'm not sure exactly why you're doing this, but it looks like you have code here specifically to be used to make unit testing work. This is typically considered bad form, your code shouldn't need to know anything about the environment it is running in (production/testing)